static/game/hero.js

```diff
@@ -133,6 +133,7 @@ export class Hero extends Creature {
   }
 
   destroy() {
+    this.submitScore();
```

Comment:
can this instead be triggered by the appearance of the death screen? in the future, we might destroy and recreate the hero for weird reasons (like a level transition? i don't know right now. we might even have multiple lives)

Reply:
Fixed in f52e7a2